CHANGELOG.md

```diff
@@ -169,6 +169,7 @@
 - PR #5825 Enable ORC statistics generation by default
 - PR #5832 Make dictionary_wrapper constructor from a value explicit
 - PR #5833 Pin `dask` and `distributed` version to `2.22.0`
+- PR #5853 Change default ctor declaration for fixed-point to use `=default`
```

Comment:
```suggestion
- PR #5853 Change default ctor declaration for `fixed_point` to use `= default`
```

Reply:
PS - if you mark as `[skip ci]` and commit suggestion this won't re-trigger CI :)